README.md

```diff
@@ -349,7 +349,7 @@ git commit -am "My changes"
 git push myrepo mybranch
 ```
 
- - Everyday better pull from the main repo(origin) and sync your repo with it.
+ - You should pull from the main repo(origin) regulary and sync it your repo.
```

Comment:
Nice finding - accepted . Please join our Jira at https://openbdre.atlassian.net/ . You can report bugs/vote or ask for features or work on other things. Also join our googlegroup. It's bdre@googlegroups.com (public - no approvals)